test/sagemaker_tests/tensorflow/tensorflow2_training/resources/mnist/mnist_custom.py

```diff
@@ -15,9 +15,17 @@
 import os
 import json
 import tensorflow as tf
-import tensorflow_io as tfio
+
+tf_major, tf_minor, _ = tf.__version__.split('.')
+if int(tf_major) > 2 or (int(tf_major) == 2 and int(tf_minor) >= 6):
```

Comment:
I believe `packaging` should already be installed - you could do something like this, which might be a little more straightforward and clear

```
from packaging.version import Version

if Version(tf.__version__) >= Version("2.6.0"):
    import tensorflow_io as tfio
```